main.go

```diff
@@ -206,10 +206,19 @@ func overrideDefaults(defaults *Options, args []string) {
 func parseDefaultConfigs(parser *flags.Parser) error {
 	app := xdg.App{Name: "yab"}
 
-	configFile := app.ConfigPath("defaults.ini")
-	if _, err := os.Stat(configFile); err != nil {
-		// No defaults file to read
-		return nil
+	// Find the best config path to use, preferring the user's config path and
```

Comment:
AFAIK the current parser doesn't allow this (although obviously it's possible to do in principle).